src/test/ui/home/testhome.py

```diff
@@ -73,10 +75,125 @@ def test_sender_overlay(self):
         except NoSuchElementException:
             print("*** element not found** ")
             self.assertTrue(False, "element not found")
-                            
-   
 
-    
+    def test_flyer_overlay(self):
+        self.driver.get('http://ishipsafe.com')
```

Comment:
unittest.Testcase also supports method level setup/cleanup. Can we pls move this first line which is common across all to the setup method?